language/gpt-j/main.py

```diff
@@ -41,12 +35,13 @@ def get_args():
                         help="user config for user LoadGen settings such as target QPS")
     parser.add_argument("--max_examples", type=int, default=None,
                         help="Maximum number of examples to consider (not limited by default)")
-    parser.add_argument("--model_script_path", default="./quantization/model_script/Qlevel1_RGDA0-W8A16-PTQ.yaml", help="")
-    parser.add_argument("--use_mcp", action="store_true", default=False, help="use mcp to quantize the model")
-    parser.add_argument("--recalibrate", action="store_true", default=False, help="load already existing quantization metadata")
+    parser.add_argument("--quant_config_path", help="a config for model quantization")
+    parser.add_argument("--quant_param_path", help="quantization parameters for calibraed layers")
+    parser.add_argument("--quant_format_path", help="quantization specifications for calibrated layers")
+    parser.add_argument("--quantize", action="store_true", help="quantize model using ModelComPressor(MCP)")
```

Comment:
```suggestion
    parser.add_argument("--quantize", action="store_true", help="quantize model using ModelComPressor (MCP)")
```

그리고, MCP가 ModelComPressor의 약자가 맞는지 알고리듬 팀에 확인 바랍니다.

Reply:
https://github.com/furiosa-ai/inference/commit/edf2c93d9baaec158d91575194d2e9170ca6a828 반영했습니다.